### ISSUE TITLE
Missing page Items with Frio

### ISSUE THREAD

**Issue by astifter** [friendica/friendica#4921]
I have sometimes (but I do not know when) the problem that not all elements of the page are rendered (see screenshot) when I use frio.
I make sure latest develop branch is checked out and remove everything except `.htconfig.php`. I also make sure that all files are owned by www-data and 0644 for files and 0755 for folders is set. Then I run `sudo -u www-data ./bin/composer.phar install`.

The login page looks and works fine but when I actually login: 
![screenshot-2018-4-23 my friend network network](https://user-images.githubusercontent.com/206846/39154894-3def9494-4750-11e8-945d-ad82d60fd8ee.png)

When I change the default theme from `frio` to `vier` in `.htconfig.php` and reload the page immediately everything works fine:
![screenshot-2018-4-23 my friend network network 1](https://user-images.githubusercontent.com/206846/39155256-8c2bbfec-4751-11e8-8eef-eee22619ae2f.png)

System: current develop with nginx and mariasql.

**Comment by MrPetovan** [comment]
Can you submit your `.htconfig.php` minus the database credentials?

**Comment by astifter** [comment]
I'm on my mobile now but its the one created by the regular install process, I have not made any manual changes except for switching themes. (But will hand it in tomorrow.)

**Comment by MrPetovan** [comment]
This is most peculiar. It definitely is the network page with the sorting tabs, but the header doesn't show any authentication, and the network page isn't supposed to be reachable as an anonymous user.

**Comment by astifter** [comment]
Okay, I have set up a fresh Virtualbox (Debian 9, 32bit) and installed necessary software and friendica as per install documentation. Install log is attached:
[install_friendica.log](https://github.com/friendica/friendica/files/1944308/install_friendica.log)

I had a problem that the automated rewrite detection via <url>/install/testrewrite did not work (despite being properly configured) and I had to comment out the URL rewrite check.

After configuring the website via the webforms I was logged in and ready to go:
![screenshot-2018-4-24 my friend network network 1](https://user-images.githubusercontent.com/206846/39210273-7cc09ba6-4808-11e8-9e78-3228453875eb.png)

When I then simply modify `.htconfig.php` to use frio instead of vier the site breaks down:
![screenshot-2018-4-24 my friend network network](https://user-images.githubusercontent.com/206846/39210305-9ac5ef5c-4808-11e8-93bc-284e52ff1fad.png)

Changing back and reloading the site everything works fine again.

`.htconfig.php` is attached as per request as well. 
[htconfig.php.txt](https://github.com/friendica/friendica/files/1944338/htconfig.php.txt)

**Comment by mark4409** [comment]
I've had this a couple of times too. I usually fix it by manually typing in the url for admin and then clicking reload current theme

**Comment by MrPetovan** [comment]
That's so weird, I never got this in a year and a half.

**Comment by astifter** [comment]
@mark4409 That indeed does the trick. But what is happening?

**Comment by MrPetovan** [comment]
Do you have anything in your PHP log or Friendica log that could help us debug?

**Comment by tobiasd** [comment]
There is a button at the top of the admin panel theme page to reload all active themes to make this easier actually.

**Comment by astifter** [comment]
Okay, I figured it out: after installing Friendica the themes are available and listed in the admin dialog as being selectable but the Theme::install(<name>) was never called for the theme. 
So in src/admin.php:1122 the theme is set in the config but can not be used properly. I have inserted a call to Theme::install() right after that line and that works but this does not seem to be the proper place to put it. Any suggestions where to ::install() those themes?
